src/pudl/helpers.py

```diff
@@ -2185,3 +2185,305 @@ def retry(
             )
             time.sleep(delay)
     return func(**kwargs)
+
+
+def standardize_phone_column(df: pd.DataFrame, columns: list[str]) -> pd.DataFrame:
+    """Standardize phone numbers in the specified columns of the DataFrame.
+    US numbers: ###-###-####
+    International numbers with the international code at the beginning.
+    Numbers with extensions will be appended with "x#".
+    Non-numeric entries will be returned as np.nan. Entries with fewer than
+    10 digits will be returned with no hyphens.
+
+    Args:
+    df: The DataFrame to modify.
+    columns: A list of the names of the columns that need to be standardized
+    Returns:
+    The modified DataFrame with standardized phone numbers in the same column.
+    """
```

Comment:
This docstring won't render correctly in the docs. Check how it looks in the built API docutmentation.

```suggestion
    """Standardize phone numbers in the specified columns of the DataFrame.
    
    US numbers: ###-###-####
    International numbers with the international code at the beginning.
    Numbers with extensions will be appended with "x#".
    Non-numeric entries will be returned as np.nan. Entries with fewer than
    10 digits will be returned with no hyphens.

    Args:
        df: The DataFrame to modify.
        columns: A list of the names of the columns that need to be standardized

    Returns:
        The modified DataFrame with standardized phone numbers in the same column.
    """
```

Reply:
Updated